whois-api/src/test/java/net/ripe/db/whois/api/httpserver/AbstractHttpsIntegrationTest.java

```diff
@@ -23,7 +23,7 @@ public static void disableHttps() {
     }
 
     public int getSecurePort() {
-        return jettyBootstrap.getSecurePort();
+        return jettyBootstrap.getClientAuthPort();
```

Comment:
Don't do this here - as the *client auth* connector will be used. Use the secure port for Https tests and override this method in `AbstractClientCertificateIntegrationTest` instead

Reply:
Or instead use `getClientCertificatePort()` and don't override or change  `getSecurePort()`

Reply:
yes, I was testing something and I left that there by mistake.
It is already in AbstractClientCertificationIntegrationTests
Just forgot to turn that return back to the previous value